aea/cli/test.py

```diff
@@ -452,16 +452,39 @@ def load_package(
     )
 
 
-def _call_pytest_in_subprocess(args: List[str]) -> NoReturn:
+def _call_pytest_in_subprocess(
+    package_dir: Path,
+    aea_project_path: Optional[Path],
+    packages_dir: Optional[Path],
+    skip_consistency_check: bool,
+    args: List[str],
+) -> NoReturn:
     """Perform call of pytest in a subprocess with pytest return code."""
+    load_package(package_dir, aea_project_path, packages_dir, skip_consistency_check)
```

Comment:
This means we're loading packages at two separate places now, please open an issue to address this in the next release